runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```diff
@@ -413,13 +413,21 @@ boolean valueSearch(final int itemId, final String str)
 		final int qty = itemQuantities.count(itemId);
 		final long gePrice = (long) itemManager.getItemPrice(itemId) * qty;
 		final long haPrice = (long) itemComposition.getHaPrice() * qty;
+		final boolean isPlaceholder = itemComposition.getPlaceholderTemplateId() == 14401;
```

Comment:
use `NullItemID.NULL_14401`

Reply:
or even better, just check `!= -1` I think like how ItemManager does.

Reply:
Ok, this change + the others have been made.